eclipse/src/saros/ui/messages.properties

```diff
@@ -106,7 +106,7 @@ PersonalizationPreferencePage_visible_information_group_text=Visual Information
 
 ConfigurationWizard_account_created_text=Your XMPP Account has already been created\n and added to your Saros configuration.\n\n Do you really want to cancel?
 ConfigurationWizard_account_created=Account Created
-ConfigurationWizard_info_already_created_account=If you already have an XMPP account you can use it right away.
+ConfigurationWizard_info_already_created_account=If you already have an XMPP account you can use it right away.\nOtherwise you can register an account at our website: <a href="https://saros-con.imp.fu-berlin.de:5280/register/">https://saros-con.imp.fu-berlin.de:5280/register/</a>
```

Comment:
This link does not point to the registration page but to the general "account management" page (where you can also delete accounts or change passwords). The link to the "create account" page is `https://saros-con.imp.fu-berlin.de:5280/register/new`. Was that done on purpose?

Reply:
I think linking "account management" would be ok, but changed it.